### PR TITLE
Html css popup

### PR DESCRIPTION
**Pull request by ASTkach**
###  html-css-popup

[Demo](https://astkach.github.io/hooli-style-popup/) |
[Code base](https://github.com/ASTkach/hooli-style-popup)

The code is submitted in a dedicated feature branch.

Only code files are submitted.

Please, review.

### CODE REVIEW

File: submissions/ASTkach/popup/index.html [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+                                    <input type="checkbox" class="nav__checkbox nav__checkbox--popup" tabindex="4" />
+                                    <div class="nav__content">
+                                        <input type="checkbox" class="nav__checkbox nav__checkbox--more-less" name="name" id="more-less" tabindex="19" />
+                                        <label class="nav__label" for="more-less"
+                                            ><span class="nav__label--more">More</span><span class="nav__label--less">Less</span></label
+                                        >
+                                        <ul class="nav__sub-list">
+                                            <li class="nav__sub-item">
+                                                <a class="nav__sub-link" href="#" target="_blank" rel="nofollow noopener" tabindex="7">
+                                                    <img src="img/icons/calendar.png" alt="calendar" class="nav__img" width="886" height="709" />
```
Comment (A-Ostrovnyy) [inline submissions/ASTkach/popup/index.html:37]: Read provided article to find out how and when to populate alt attribute value [Как правильно написать alt-текст](https://htmlacademy.ru/blog/articles/alt-text)

File: submissions/ASTkach/popup/index.html [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+                    <nav class="nav">
+                        <ul class="nav__list">
+                            <li class="nav__item nav__item--hidden">
+                                <a class="nav__link" href="#" target="_blank" rel="nofollow noopener" tabindex="2">Mail</a>
+                            </li>
+                            <li class="nav__item nav__item--hidden">
+                                <a class="nav__link" href="#" target="_blank" rel="nofollow noopener" tabindex="3">Photo</a>
+                            </li>
+                            <li class="nav__item">
+                                <div class="nav__popup _icon-popup-button">
```
Comment (A-Ostrovnyy) [inline submissions/ASTkach/popup/index.html:27]: When the popup is open focus must go to the first interactive element inside the popup. In our case when the user opens popup and presses the` Tab` key focus must go to the first link in the popup

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Awesome!

File: submissions/ASTkach/popup/index.html (hunk 1 above)
Comment (ASTkach) [reply submissions/ASTkach/popup/index.html:37]: I don't really understand what alt-text should be written for such simple pictures. "yellow folder with documents"? "piece of map with geolocation"?

Comment (A-Ostrovnyy) [reply submissions/ASTkach/popup/index.html:37]: `alt` text must be provided for content images. The icon is a decorative image so `alt` attribute should be empty)

Comment (ASTkach) [reply submissions/ASTkach/popup/index.html:37]: omg... Thanks!

File: submissions/ASTkach/popup/index.html [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+                                        </ul>
+                                    </div>
+                                </div>
+                            </li>
+                        </ul>
+                    </nav>
+                    <div class="header__user">
+                        <a href="#" class="header__notification _icon-bell" target="_blank" rel="nofollow noopener" tabindex="24"></a>
+                        <a href="#" class="header__profile _icon-logo" target="_blank" rel="nofollow noopener" tabindex="25">
+                            <!-- <img src="img/logo/logo.svg" alt="profile" class="header__icon" width="24" height="24" /> -->
```
Comment (A-Ostrovnyy) [inline submissions/ASTkach/popup/index.html:152]: Avoid commented code in PR

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Did you update your demo? Because the order of interactive elements is the same [watch video](https://share.vidyard.com/watch/YvYAaTFLY5h6wqyHH4G4k2?)

File: submissions/ASTkach/popup/index.html (hunk 3 above)
Comment (ASTkach) [reply submissions/ASTkach/popup/index.html:152]: Already updated. Sorry.

Comment (A-Ostrovnyy) [reply submissions/ASTkach/popup/index.html:152]: Did you forget to update the current PR?)

Comment (ASTkach) [reply submissions/ASTkach/popup/index.html:152]: omg... push wrong index.html ((( One minute.

Comment (ASTkach) [reply submissions/ASTkach/popup/index.html]: done

File: submissions/ASTkach/popup/index.html [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+                                                </a>
+                                            </li>
+                                            <li class="nav__sub-item">
+                                                <a class="nav__sub-link" href="#" target="_blank" rel="nofollow noopener" tabindex="9">
+                                                    <img src="img/icons/files.png" alt="" class="nav__img" width="886" height="531" />
+                                                    <p class="nav__text">Files</p>
+                                                </a>
+                                            </li>
+                                            <li class="nav__sub-item">
+                                                <a class="nav__sub-link" href="#" target="_blank" rel="nofollow noopener" tabindex="10">
```
Comment (A-Ostrovnyy) [inline submissions/ASTkach/popup/index.html:66]: No need to increment `tabindex` for sibling elements. You can use the same value. Check/play with [this demo](https://codepen.io/a-ostrovnyy/pen/ZExXXvq)

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Good progress!

**Review by A-Ostrovnyy — APPROVED** [review]
Well done!

**Pull request merged**